README.md

```diff
@@ -120,6 +120,30 @@ and tactics specialized for monadic programs (see
 
 A tutorial for the Lean backend is available [here](./tests/lean/Tutorial.lean).
 
+## Quick start for Nix users
+
+Assuming Nix is installed, with a support for Flakes (`*`):
+
+```console
+$ nix run github:aeneasverif/aeneas#charon -L
+$ # The default package for Aeneas is Aeneas.
+$ nix run github:aeneasverif/aeneas -L -- -backend your_preferred_backend your_llbc.file
+```
+
+To regenerate the extraction, just run step 2 and step 3 again.
+
+Sometimes, Charon and Aeneas can become desynchronized, changes of the LLBC on
+Charon side will not have been taken into account on Aeneas side yet. We try to
+avoid this as much but if you are using Nix, this is the simple way to avoid
+that problem. https://github.com/AeneasVerif/charon/issues/35 is related to that matter.
```

Comment:
We now have version numbers. This paragraph still makes sense but you can mention the new behavior.

Reply:
Actually, this won't happen if the users uses the charon exported by aeneas, so you can remove the paragraph.

Reply:
Removed.